src/control/conf.c

```diff
@@ -832,6 +832,25 @@ gchar* dt_conf_expand_default_dir(const char *dir)
 
   return normalized_path;
 }
+void dt_conf_save(dt_conf_t *cf)
```

Comment:
ok, but I suppose we have another function (or piece of code) somewhere to save the conf. Maybe we could reuse it or call this version... Whatever but here I bet we have some duplicate code.